mongodbatlas.go

```diff
@@ -134,7 +134,7 @@ func (m *MongoDBAtlas) RevokeUser(ctx context.Context, statements dbplugin.State
 		return err
 	}
 
-	_, err = client.DatabaseUsers.Delete(ctx, m.ProjectID, username)
+	_, err = client.DatabaseUsers.Delete(ctx, "admin", m.ProjectID, username)
```

Comment:
Maybe we could get the database name from creation statements here too. I don't know if it's the case.

Reply:
Right now it is now much of an issue but in the future, maybe some other databases can be supported too.